challenge-sizebay/src/Components/ToDoList/ToDoList.jsx

```diff
@@ -0,0 +1,80 @@
+import React, { useContext } from "react";
+import { GlobalContext } from "../../Contexts/GlobalContext";
+import {
+  List,
+  ListItem,
+  RemoveButton,
+  CompleteButton,
+  ButtonsContainer,
+  EmptyListMessage,
+  ClearFilters,
+} from "./style";
+import check from "../../assets/check.png";
+import remove from "../../assets/remove.png";
+import { Tooltip } from "react-tooltip";
+
+const ToDoList = () => {
+  const { states, setters } = useContext(GlobalContext);
+
+  const returnList = () => {
+    return states.listToShow === "pending" ? states.pending : states.done;
+  };
+
+  const completeTask = (task) => {
+    const newPendingList = states.pending.filter((pendingTask) => {
+      return pendingTask !== task;
+    });
+    setters.setPending(newPendingList);
+    setters.setDone([...states.done, task]);
+  };
+
+  const deleteTask = (task) => {
+    const newPendingList = states.pending.filter((pendingTask) => {
+      return pendingTask.toLowerCase() !== task.toLowerCase();
+    });
+    setters.setPending(newPendingList);
+  };
+
+const clearFilters = () => {
+  setters.setListToShow('')
+}
+  return (
+    <List>
+      {returnList().length === 0 ? (
+        <EmptyListMessage>
+          {states.listToShow && `There are no items marked as ${states.listToShow}. Clear the filter here to`}
+          <ClearFilters onClick={clearFilters}> see all items</ClearFilters>
+        </EmptyListMessage>
+      ) : (
+        returnList()
```

Comment:
ref: after changing the code mentioned on https://github.com/sizebay/frontend-challenge/pull/18/files#diff-20c62e36dedf4d52081a62fd77197526552be193dfdbbebc9d337fb1111f0e43R19-R21, you won't need to implicitly execute the function, but declare it as a variable instead (e.g `returnList.length`)

Reply:
Done